Core.Telemetry/OpenTelemetryClient.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.Extensions.Logging;
+using OpenTelemetry;
+using OpenTelemetry.Exporter.Geneva;
+using OpenTelemetry.Logs;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.CloudMine.Core.Telemetry
+{
+    public class OpenTelemetryClient : ILoggerProvider
```

Comment:
Are ILoggerProvider and ILoggerFactory coming from OT library or did you create these interfaces?

Reply:
Both `ILoggerFactory` and `ILoggerProvder` come from Microsoft.Extensions.Logging these are standard interfaces used for logging injection in dotnet. OpenTelemetry provides implementations of these interfaces for OT logs.